File: ChildApp/src/app/model/Entities.ts
export class SelectList {
    Value: string; Text: string;
}
export class ItemList {
    Id: string;
    ChildID: number;
    UpdatedBy: string;
    Remarks: string;
}
export class SearchFilter {
    Gender: SelectList[];
    Type: SelectList[];
    ChildRelation: SelectList[];
    Status: SelectList[];
}
export class ChildDetails {
ChildID: number;
ChildLastName: string;
ChildFirstName: string;
ChildGender: number;
ChildBirthDate: string;
ChildStatus: number;
ChildAddress: string;
ChildType: number;
Parent1LastName: string;
Parent1FirstName: string;
Parent1Private: string;
Parent1Gender: number;
Parent1ChildRelation: number;
Parent1Address: string;
Parent1Unit: string;
Parent1City: string;
Parent1Province: string;
Parent1PostalCode: string;
Parent1HomePhone: string;
Parent1WorkPhone: string;
Parent1CellPhone: string;
Parent1Email: string;
Parent1PrimaryEmail: string;
Parent1Comments: string;
Parent1SpecialCustody: string;
Parent2LastName: string;
Parent2FirstName: string;
Parent2Private: string;
Parent2Gender: number;
Parent2ChildRelation: number;
Parent2Address: string;
Parent2Unit: string;
Parent2City: string;
Parent2Province: string;
Parent2PostalCode: string;
Parent2HomePhone: string;
Parent2WorkPhone: string;
Parent2CellPhone: string;
Parent2Email: string;
Parent2PrimaryEmail: string;
Parent2Comments: string;
Parent2SpecialCustody: string;
SiblingName: string;
SiblingAge: number;
SiblingGender: number;
Contact1Name: string;
Contact1Phone: string;
Contact1Email: string;
Contact1Address: string;
  Contact1Relationship: number;
Contact2Name: string;
Contact2Phone: string;
Contact2Email: string;
Contact2Address: string;
  Contact2Relationship: number;
Contact3Name: string;
Contact3Phone: string;
Contact3Email: string;
Contact3Address: string;
  Contact3Relationship: number;
Contact4Name: string;
Contact4Phone: string;
Contact4Email: string;
Contact4Address: string;
  Contact4Relationship: number;

}

export class ResponseMessage {
    ID: number;
    Message: string;
}
export class RequestParams {
    ChildID: number;
    ChildFirstName: string;
    ChildLastName: string;
}
export class dealdocument {
    DocumentId: number;
    ChildID: number;
    Document: string;
    FileSize: number;
    EnteredBy: string;
  EnteredOn: string;
}

File: API/API/Models/Entities.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace API.Models
{
    public class Entities
    {
    }
    public class MyResponse
    {
        public bool Status { get; set; }
        public string Message { get; set; }
    }
    public class SelectList
    {
        public string Value { get; set; }
        public string Text { get; set; }
    }
    public class ItemList
    {
        public string Id { get; set; }
        public int   ChildID { get; set; }
        public string UpdatedBy { get; set; }
        public string Remarks { get; set; }
    }
    public class SearchFilter
    {
        public List<SelectList> Gender { get; set; }
        public List<SelectList> Type { get; set; }
        public List<SelectList> ChildRelation { get; set; }
        public List<SelectList> Status { get; set; }
    }


    public class ResponseMessage
    {
        public int ID { get; set; }
        public string Message { get; set; }
    }
    public class RequestParams
    {
  
        public int ChildID { get; set; }
        public string ChildFirstName { get; set; }
        public string ChildLastName { get; set; }
 
    }
   
  
    public class ChildDocument
    {
        public int DocumentId { get; set; }
        public int ChildId { get; set;}
        public string Document { get; set; }
        public int FileSize { get; set; }
        public string EnteredBy { get; set; }
        public string EnteredOn { get; set; }
    }

    public class Childdetails
    {
public int ChildID { get; set; }
public string ChildLastName { get; set; }
public string ChildFirstName { get; set; }
        public int ChildGender { get; set; }
        public string ChildBirthDate { get; set; }
        public int ChildStatus { get; set; }
        public string ChildAddress { get; set; }
        public int ChildType { get; set; }
        public string Parent1LastName { get; set; }
        public string Parent1FirstName { get; set; }
        public string Parent1Private { get; set; }
        public int Parent1Gender { get; set; }
        public int Parent1ChildRelation { get; set; }
        public string Parent1Address { get; set; }
        public string Parent1Unit { get; set; }
        public string Parent1City { get; set; }
        public string Parent1Province { get; set; }
        public string Parent1PostalCode { get; set; }
        public string Parent1HomePhone { get; set; }
        public string Parent1WorkPhone { get; set; }
        public string Parent1CellPhone { get; set; }
        public string Parent1Email { get; set; }
        public string Parent1PrimaryEmail { get; set; }
        public string Parent1Comments { get; set; }
        public string Parent1SpecialCustody { get; set; }
        public string Parent2LastName { get; set; }
        public string Parent2FirstName { get; set; }
        public string Parent2Private { get; set; }
        public int Parent2Gender { get; set; }
        public int Parent2ChildRelation { get; set; }
        public string Parent2Address { get; set; }
        public string Parent2Unit { get; set; }
        public string Parent2City { get; set; }
        public string Parent2Province { get; set; }
        public string Parent2PostalCode { get; set; }
        public string Parent2HomePhone { get; set; }
        public string Parent2WorkPhone { get; set; }
        public string Parent2CellPhone { get; set; }
        public string Parent2Email { get; set; }
        public string Parent2PrimaryEmail { get; set; }
        public string Parent2Comments { get; set; }
        public string Parent2SpecialCustody { get; set; }
        public string SiblingName { get; set; }
        public int SiblingAge { get; set; }
        public int SiblingGender { get; set; }
        public string Contact1Name { get; set; }
        public string Contact1Phone { get; set; }
        public string Contact1Email { get; set; }
        public string Contact1Address { get; set; }
        public int Contact1Relationship { get; set; }
        public string Contact2Name { get; set; }
        public string Contact2Phone { get; set; }
        public string Contact2Email { get; set; }
        public string Contact2Address { get; set; }
        public int Contact2Relationship { get; set; }
        public string Contact3Name { get; set; }
        public string Contact3Phone { get; set; }
        public int Contact3Email { get; set; }
        public string Contact3Address { get; set; }
        public string Contact3Relationship { get; set; }
        public string Contact4Name { get; set; }
        public string Contact4Phone { get; set; }
        public int Contact4Email { get; set; }
        public string Contact4Address { get; set; }
        public string Contact4Relationship { get; set; }

    }

    }